RoddyCore/src/de/dkfz/roddy/config/ToolEntry.java

```diff
@@ -217,6 +219,39 @@ public ToolStringParameter clone() {
         }
     }
 
+    public static class ToolFileParameterCondition {
+
+        private String condition;
+
+        private Boolean pureBoolean;
+
+        public ToolFileParameterCondition(boolean pureBoolean) {
+            this.pureBoolean = Boolean.valueOf(pureBoolean);
+        }
+
+        public ToolFileParameterCondition(String condition) {
+            if(condition.startsWith("conditional:"))
```

Comment:
This is something different, you cannot pass it via cli! And it does not set a type. It just names the variable, which is used to define value of check.

<output ... check="condition:variablename" />